### PR TITLE
Fix typo in command

### PR DESCRIPTION
**Pull request by josiemundi**
Fixed a typo

### CODE REVIEW

File: docs/install/any-kubernetes-cluster.md [hunk 1]
```diff
@@ -219,6 +219,37 @@ The following commands install Istio and enable its Knative integration.
...
+1. Fetch the External IP or CNAME:
+
+   ```bash
+   kubectl --namespace kong get service kong-proxy
+   ```
+
+   Save this for configuring DNS below.
+
+{{< /tab >}}
+
```
Comment (markusthoemmes) [inline docs/install/any-kubernetes-cluster.md:252]: Is this supposed to be part of this PR? The title indicates otherwise.

Comment (josiemundi) [reply docs/install/any-kubernetes-cluster.md:252]: No, sorry I just realised that there was another commit in there. Mine was only to change the -n istio typo.

Comment (mpetason) [reply docs/install/any-kubernetes-cluster.md:252]: Maybe close this and open a new one without the other commit?

Comment (josiemundi) [reply docs/install/any-kubernetes-cluster.md:252]: @mpetason I have opened a new PR for this.

**Pull request closed without merging**